nginx/ngx_http_js_module.c

```diff
@@ -1540,6 +1540,9 @@ ngx_http_js_variable_set(ngx_http_request_t *r, ngx_http_variable_value_t *v,
     }
 
     if (rc == NGX_DECLINED) {
+        ngx_log_error(NGX_LOG_ERR, r->connection->log, 0,
+                      "no \"js_import\" directives found for \"js_set\" \"%V\""
+                      " in the currect scope", fname);
```

Comment:
The `\"js_set\" \"%V\"` part looks a bit confusing, as if a word is missing there. Perhaps it should be `\"js_set\" function \"%V\"`.

Also, "currect" -> "current".